server/runtime/src/main/java/org/infinispan/server/configuration/ServerConfigurationParser.java

```diff
@@ -736,27 +734,27 @@ private void parseLdapIdentityMapping(ConfigurationReader reader, LdapIdentityMa
             case SEARCH_RECURSIVE:
                identityMapBuilder.searchRecursive(Boolean.valueOf(value));
                break;
+            case SEARCH_TIME_LIMIT:
+               identityMapBuilder.searchTimeLimit(Integer.valueOf(value));
```

Comment:
avoid boxing&unboxing
```suggestion
               identityMapBuilder.searchTimeLimit(Integer.parseInt(value));
```

Reply:
Fixed